Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/Helper/S3Helper.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import AblyAssetTrackingPublisher
+import Amplify
+import AWSCognitoAuthPlugin
+import AWSS3StoragePlugin
+
+class S3Helper {
+        
+    init() {
+        do {
+            try Amplify.add(plugin: AWSCognitoAuthPlugin())
+            try Amplify.add(plugin: AWSS3StoragePlugin())
+            try Amplify.configure()
+            print("Amplify configured with Auth and Storage plugins")
+        } catch {
+            print("Failed to initialize Amplify with \(error)")
```

Comment:
A note to all the prints in this PR - we have a `PublisherLogger` that we use in the publisher factory method in `CreatePublisherViewModel`. We should probably use it instead of those prints (for example by using it as a singleton).

Reply:
done